### PR TITLE
Allow overriding system-reserved shortcuts

### LINKED ISSUE #33: Allow some system-reserved keyboard shortcuts that are not used?
great package. i'm new to macOS and Swift development and got things working painlessly 👍🏼️

wondering though whether there's a way i could see why some shortcuts are said to be used system-wide while they're not registered in the macOS keyboard shortcuts settings. any idea? i.e. "Command + Esc". some other apps like 1Password give the same warning message, but some others like Dash allow the use of those shortcuts. so i guess there's an implementation difference (which is fine). just trying to understand what those system-wide shortcuts that are not registered in the macOS settings are doing and if i can change them. thanks!

### PR DESCRIPTION
**Pull request by tofumatt**
Fix #33.

Updated version of #125 with the linting errors fixed, and "OK" as the default option in the dialog.

Works as-expected in my app:

https://github.com/sindresorhus/KeyboardShortcuts/assets/90871/d155611d-d1b2-4209-97b0-490cd326a1b0

### CODE REVIEW

File: Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -3,3 +3,5 @@
 "keyboard_shortcut_used_by_menu_item" = "This keyboard shortcut cannot be used as it’s already used by the “%@” menu item.";
 "keyboard_shortcut_used_by_system" = "This keyboard shortcut cannot be used as it’s already a system-wide keyboard shortcut.";
 "keyboard_shortcuts_can_be_changed" = "Most system-wide keyboard shortcuts can be changed in “System Settings › Keyboard › Keyboard Shortcuts”.";
+"force_use_shortcut" = "Use Anyway";
+"ok" = "OK";
```
Comment (tofumatt) [inline Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings:7]: I'm not very familiar with localisations in Swift/Xcode projects, please let me know if there's a built-in default for "OK"/"yes" dialogs provided by the localisation framework; I'm happy to use that if there is 🙂

Comment (sindresorhus) [reply Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings:7]: There's nothing built-in.

Would be great if you could fill in "OK" for the existing localizations though: https://applelocalization.com/macos?q=OK&l=English&l=French&l=German&l=Italian&l=Japanese&l=Spanish

**Comment by sindresorhus** [comment]
What keyboard combination do you use to test the dialog (to trigger it)?

File: Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (tofumatt) [reply Sources/KeyboardShortcuts/Localization/en.lproj/Localizable.strings:7]: Cool, I've added them for all languages in the repo, based on the text in Apple's docs. Thanks! 🙂

**Comment by tofumatt** [comment]
I used "Command + Escape", which is actually my usual keyboard shortcut for an app I'm transitioning to use `KeyboardShortcuts` from another library, so at first was very confused why it wouldn't work 😅

File: Sources/KeyboardShortcuts/RecorderCocoa.swift [hunk 2]
```diff
@@ -283,19 +283,26 @@ extension KeyboardShortcuts {
...
-				guard !shortcut.isTakenBySystem else {
+				if shortcut.isTakenBySystem {
 					self.blur()
 
-					NSAlert.showModal(
+					let modalResponse = NSAlert.showModal(
 						for: self.window,
 						title: "keyboard_shortcut_used_by_system".localized,
 						// TODO: Add button to offer to open the relevant system settings pane for the user.
-						message: "keyboard_shortcuts_can_be_changed".localized
```
Comment (chatobn) [inline Sources/KeyboardShortcuts/RecorderCocoa.swift:293]: - mensaje:"@luzmary"
- [ ]

**Review by chatobn — CHANGES_REQUESTED** [review]

**Pull request merged**